scripts/build-bpf.sh

```diff
@@ -1,10 +1,4 @@
 #!/usr/bin/env bash
-#
-# Build given bpf makefile dir (./program by default):
-#   ~/pyth-client$ ./scripts/build-bpf.sh
-#   ~/pyth-client/program$ ../scripts/build-bpf.sh .
-#   ~/$ ./pyth-client/scripts/build-bpf.sh ./serum-pyth/program
-#
 
```

Comment:
This comment felt outdated given that now we're calling cargo build from the root folder.